src/redux/actions/actions.js

```diff
@@ -1876,6 +1973,79 @@ export const getReportsAuthors = (pmids) => {
         })
 }
 
+// Get authors of publication
+export const createAdminUser = (userData) =>{
+    return fetch(`/api/db/admin/users/addEditUser`, {
```

Comment:
can we separate apis as 
1. /api/db/admin/users/edit
2. /api/db/admin/users/add
Because addEdit together is confusing

Reply:
I understood. But, it is a few lines of code, I mean only condition check. is it necessary to add another file?